crates/rbuilder/src/bin/misc-relays-slot.rs

```diff
@@ -1,3 +1,5 @@
+//! Helper app to get information from a landed block from the relays.
+//! Take no configuration since it uses a hardcoded list of well known relays ([`rbuilder::mev_boost::RELAYS`])
```

Comment:
Takes

Reply:
see my comment above, also here we could have "hardcoded list of relays" (without "well known").